app/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM debian:10
+
+ENV SUMMARY="An image for the connectivity intake app" \
+    DESCRIPTION="This image contains the compiled connectivity node app"
+
+LABEL summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      io.k8s.display-name="Connectivity Intake" \
+      io.openshift.tags="connectivity,node" \
+      name="connectivity-intake" \
+      version="1" \
+      maintainer="Bryan Krauss <bryan@button.is>"
+
+RUN apt-get update && \
+    apt-get install -y git gnupg curl && \
+    apt-get clean
+
+ENV USER_ID=1001
+ENV APP_HOME=/root
+ENV HOME=/root
+
+WORKDIR ${APP_HOME}
+
+COPY app/ ${APP_HOME}/
+
+RUN git clone https://github.com/asdf-vm/asdf.git ${APP_HOME}/asdf --branch v0.8.1
+RUN cd ${APP_HOME}/asdf && git checkout v0.8.1
+ENV BASH_ENV="${APP_HOME}/asdf/asdf.sh"
+# Because asdf is loaded via BASH_ENV, all commands using adsf need to be executed using /usr/bin/env bash -c
+SHELL ["/usr/bin/env", "bash", "-c"]
+
+COPY .tool-versions ${APP_HOME}/.tool-versions
+# The app only needs yarn and node
+RUN sed -i -nr '/node|yarn/p' ${APP_HOME}/.tool-versions
+RUN cat ${APP_HOME}/.tool-versions | cut -f 1 -d ' ' | xargs -n 1 asdf plugin-add
+RUN asdf plugin-update --all
+
+RUN asdf install
+RUN asdf reshim
+
+COPY .bin/docker-entrypoint.sh ${APP_HOME}
+ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.2.0_amd64 /usr/local/bin/dumb-init
+RUN chmod +x /usr/local/bin/dumb-init
+ENTRYPOINT ["dumb-init", "--", "./docker-entrypoint.sh"]
```

Comment:
I was looking at the `docker-entrypoint.sh` script in the .bin folder and it appears to be something related to openshift. Is this something you think we'll need at the moment?